### ISSUE TITLE
Not able to change template ID of clusterV1 resource

### ISSUE THREAD

**Issue by jan-di** [crossplane-contrib/provider-openstack#9]
Due to the way Upjet creates parameters for resources, attributes with Terraform's `ForceNew` flag are not handled as immutable fields. See https://github.com/upbound/upjet/issues/78 for more Info.

Expected:

1. Change cluster template ID
2. -> Cluster gets recreated, as the underlaying Terraform provider has the ForceNew flag set

Actual Behaviour:

1. Change cluster template ID
2. -> Sync does not complete:
```
Message:               observe failed: cannot run plan: plan failed: Instance cannot be destroyed: Resource openstack_containerinfra_cluster_v1.mycluster has lifecycle.prevent_destroy set, but the plan calls for this resource to be destroyed. To avoid this error and continue with the plan, either disable lifecycle.prevent_destroy or reduce the scope of the plan using the -target flag.
```

**Comment by jan-di** [comment]
Support for upgrading clusters to a new cluster templated has been implemented in the Terraform provider (has yet to be released). Upgrading to this provider may solve the issue.

https://github.com/terraform-provider-openstack/terraform-provider-openstack/pull/1598

**Comment by jan-di** [comment]
Solved by #22
